### ISSUE TITLE
update app to login

### ISSUE THREAD

**Issue by ghost** [telefuel/recipe-telefuel#1]
![Screenshot_2022-12-19_22_07_40](https://user-images.githubusercontent.com/78384696/208503164-2885e86f-be63-43eb-8d88-485222cb75b8.png)

**Comment by verybrickednexus5x** [comment]
Yes, I have this exact same issue. Can anyone please help? Telegram looks the best in my opinion
